pkg/applier/applier.go

```diff
@@ -163,25 +161,29 @@ func NewSupervisor(cs *ClientSet, scope declared.Scope, syncName string, reconci
 // objects in a single namespace.
 func NewNamespaceSupervisor(cs *ClientSet, namespace declared.Scope, syncName string, reconcileTimeout time.Duration) (Supervisor, error) {
 	syncKind := configsync.RepoSyncKind
-	invObj := newInventoryUnstructured(syncKind, syncName, string(namespace), cs.StatusMode)
+	syncNamespace := string(namespace)
```

Comment:
nit: scope has a SyncNamespace method, but i guess it's not technically required here. Maybe we should combine these constructors in the future tho. They're not all that different.